### ISSUE TITLE
jpeg generation

### ISSUE THREAD

**Issue by briantjacobs** [pramsey/gdal2tilesp#4]
This script is great, but it's missing an option for setting the quality of jpeg that's being generated. Also unclear what quality is currently being used.

Also, when creating jpegs, I don't see an option to specify what color transparent edges will be rendered as (it defaults to black). My input data doesn't have any NODATA, but when tiling the edges of it, half-black tiles get rendered because it's rendering beyond my file.

**Comment by roblabs** [comment]
I agree that the jpeg quality is not one of the parameters.  At first look, it isn't clear to me how to change or add that parameter. 

But I did find this on the [gdal.org ](http://www.gdal.org/frmt_jpeg.html)site. 

`QUALITY=n: By default the quality flag is set to 75`

Again, this issue or feature would be to add another parameter for quality.  There is a [section of code](https://github.com/pramsey/gdal2tilesp/blob/e9eab7692f7348d830d280bd03f794f81bd624a6/gdal2tilesp.py#L1389) that checks for `JPEG`, and that _may_ be the right place to add the quality parameter.
